Makefile

```diff
@@ -0,0 +1,66 @@
+artifact_name       := company-documents.web.ch.gov.uk
+
+.PHONY: all
+all: build
+
+.PHONY: clean
+clean:
+	mvn clean
+	rm -f ./$(artifact_name).jar
+	rm -f ./$(artifact_name)-*.zip
+	rm -rf ./build-*
+
+.PHONY: build
+build:
+	mvn install
+
+.PHONY: test
+test: test-unit test-integration
+
+.PHONY: test-unit
+test-unit: clean
+	mvn test
+
+.PHONY: test-integration
```

Comment:
No need for these targets just yet . We can add them later when we add integration tests .